### ISSUE TITLE
Please Add Height Range Modifier

### ISSUE THREAD

**Issue by kgearhardt** [SoftFever/OrcaSlicer#533]
One feature I miss the most having switched from printing on my MK3S+ to the P1P is the lack of the height range modifier in Bambu Studio and OrcaSlicer.

Knowing nothing about the related code in PrusaSlicer or SuperSlicer, it would be great to see that code ported over to OrcaSlicer in a future release.

Cheers!

**Comment by kgearhardt** [comment]
I 3D print tap handle medallions for our keg distribution business, and part of the print is pausing to insert magnets.  On the Prusa I change the layer height from 0.2mm to 0.1mm so that I can pause after the "perfect" depth to insert the magnet, and then resume with a layer that's just 0.1mm above the magnets.  Then I'll switch back to 0.2mm heights.

It's arguably a luxury as I just printed a bunch on the P1P sliced with OrcaSlicer at 0.2mm layer heights, and it worked just fine.  I guess PrusaSlicer/SuperSlicer just spoiled me because that's what I started with.

Hope that makes sense.
